src/yb/tools/yb-admin_client.cc

```diff
@@ -1673,14 +1673,21 @@ Status ClusterAdminClient::ModifyPlacementInfo(
                                                     strings::SkipEmpty());
     if (block.size() != 3) {
       return STATUS(InvalidCommand, "Each placement info must have exactly 3 values seperated"
-          "by dots that denote cloud, region and zone. Block: " + placement_info_split[iter]
-          + " is invalid");
+          "by dots that denote cloud, region, zone and number of replicas. Block: "
+          + placement_info_split[iter] + " is invalid");
     }
+    std::vector<std::string> replica_block = strings::Split(block[2], ":",
+                                                            strings::SkipEmpty());
+    if (replica_block.size() != 2) {
```

Comment:
I think we should leave the existing behavior as is, so if its just `cloud.region.zone` then default to 1 for min_num_replicas.

Reply:
Yeah, otherwise it would be a breaking change. Will update the PR with real soon. Thanks for the review! :)